saw/llvm.py

```diff
@@ -150,13 +150,38 @@ def uniquify(x : str, used : Set[str]) -> str:
 
 
 class PointsTo:
-    def __init__(self, pointer : SetupVal, target : SetupVal) -> None:
+    def __init__(self, pointer : SetupVal, target : SetupVal, condition : Optional[CryptolTerm] = None) -> None:
         self.pointer = pointer
         self.target = target
+        self.condition = condition
 
     def to_json(self) -> Any:
         return {"pointer": self.pointer.to_json(),
-                "points to": self.target.to_json()}
+                "points to": self.target.to_json(),
+                "check points to type": {"check against": "pointer type"},
+                "condition": self.condition.to_json() if self.condition is not None else self.condition}
+
+
+class PointsToUntyped(PointsTo):
```

Comment:
Admittedly, I got lazy here and didn't document these because `PointsTo` had no docstring either. A poor excuse, I know. I'll make sure to add docstring(s) to whatever design we end up with.

Reply:
I've added a docstring to `points_to`.